include/model/GPRModel.hpp

```diff
@@ -36,80 +36,36 @@
 #include "model/Model.hpp"
 #include "model/ModelFramework.hpp"
 #include "model/ScikitLearnFramework.hpp"
-#include "statespace/StateSpace.hpp"
 
 namespace libcozmo {
 namespace model {
 
+/// This class implements a Gaussian Process Regressor(GPR), a type of
+/// multi-dimensional linear regression model.
+///
+/// This model learns the effect of an action a (i.e. the change in state). More
+/// specifically, it learns f: a -> Δs where a is
+/// [speed, aspect ratio, edge offset] and Δs is [delta distance, delta theta].
```

Comment:
hmm maybe say `a is an object oriented action and delta s is the change in the SE2 state which is represented by the distance the object moved along the action vector and the change in its orientation`